### DISCUSSION TITLE
Nothing for PyCharm?

### DISCUSSION THREAD

**Discussion by sayakpaul** [community/community#6870] [Copilot Conversations] [answered]
Seems like there isn't an installation of Copilot for PyCharm?

**Comment by yashrathi-git** [comment]
Copilot is available for pycharm. [See this](https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md#getting-started-with-github-copilot-in-jetbrains)

**Comment by sayakpaul** [comment]
Thank you!

Still unable to find it, though:

![image](https://user-images.githubusercontent.com/22957388/139195551-f057cd72-f7c6-4607-8560-4afd1899b775.png)


My PyCharm version:

![image](https://user-images.githubusercontent.com/22957388/139195619-b61b464c-78c6-48e0-a568-bce7cf4ab1a8.png)

**Reply by mdrafiqulrabin** [reply]
[Check with PyCharm versions 2021.2 and above.](https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md#1-installing-the-jetbrains-extension)

**Comment by sayakpaul** [comment]
Thanks folks. I was able to complete the process.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Copilot requires PyCharm versions 2021.2+

**Comment by LakshmankumarMukkavilli** [comment]
Not able to use Github Copilot with Pycharm.
I have pycharm 2023.1.2. Get the message
Search result not included.
Check internet connection

Ihere is no problem with Internet. I am able to other tasks that need internet.
